### ISSUE TITLE
Incorrect "engines" declaration - not actually working with node 12

### ISSUE THREAD

**Issue by Rialgar** [import-js/eslint-import-resolver-typescript#192]
Since you are importing "node:fs" and "node:path" you are requiring node.js 14.18, 16.0, or later since your version 3.0.0. Also synckit in the version you use requires node 14. However, your engines field still includes "^12.20.0".

I can file a PR, if you want, but the issue is that this is already wrong for your existing published version 3.0.0 and newer, so you may want to consider re-publishing those as well, which is a little more involved than a simple PR.

**Comment by JounQin** [comment]
You're right here, I was using `^12.20.0` before because I was thinking about ESM, but this plugin only works with commonjs, actually.

https://2ality.com/2021/12/node-protocol-imports.html#support-for-node%3A-imports

We've alreay using "node:fs" and "node:path" since 3.0, so I think it's OK to just correct the engine field.
